dataPipelines/gc_downloader/config.py

```diff
@@ -5,7 +5,8 @@
 
 SUPPORTED_FILE_EXTENSIONS = [
     ".pdf",
-    ".zip"
+    ".zip",
```

Comment:
from my knowledge we do not support zip so we should take this off

Reply:
yep from the previous experience with you me and dakota, zip isn't supported. we should keep this out until we can show in testing that zip functionality is there